<launch>
<!--    <include file="$(find rcj_pcms_base)/launch/FaceDetection.launch"/>-->
<!--    <include file="$(find rcj_pcms_base)/launch/FaceMaskDetection.launch"/>-->
<!--    <include file="$(find rcj_pcms_base)/launch/YOLODetection.launch">-->
<!--        <arg name="image_source" value="/bottom_camera_rotate/rgb/image_raw/compressed"/>-->
<!--    </include>-->
<!--    <include file="$(find rcj_pcms_base)/launch/GenderAgeDetection.launch"/>-->
    <node
        pkg="rcj_pcms_base"
        name="acp"
        type="ActionControllerNode.py"
        required="true"
        output="screen"
    />
<!--    <node-->
<!--        pkg="rcj_pcms_base"-->
<!--        name="show_face_detections"-->
<!--        type="ShowFaceDetections.py"-->
<!--        required="true"-->
<!--        output="screen"-->
<!--    />-->
    <node
        pkg="rcj_pcms_base"
        name="pose_recognition"
        type="PoseRecognition.py"
        required="true"
        output="screen"
    />
    <node
        pkg="rcj_pcms_base"
        name="manipulator_grab"
        type="ManipulatorGrab.py"
        required="false"
        output="screen"
    />
<!--    <node-->
<!--        pkg="rcj_pcms_base"-->
<!--        name="qr_code"-->
<!--        type="QrCode.py"-->
<!--        required="false"-->
<!--        output="screen"-->
<!--    />-->
</launch>
